src/db/followToDB.php

```diff
@@ -77,7 +88,19 @@ function add_follow_to_db(){
 		return $followRes;
 	}
 	
+	//returns a server path to a page
+	function get_redirect_path($value,$p_id){
+		
+		
+		switch($value){
+		/*no user*/		 		case(-3): return "../SignUpPage/signUP.php?source=post";
+		/*no user to follow*/	case(-4): return "../viewPostPage/viewPost.php?id= $p_id&source=noUserToFollow";
+		/*follow success*/ 	default: return "../viewPostPage/viewPost.php?id= $p_id";
+		}
+		return "../viewPostPage/viewPost.php?id= $p_id";
+	}
+}
 	//script
-	$output = add_follow_to_db();
+	//$output = add_follow_to_db();
```

Comment:
Why is this commented out? If this is not needed, please remove it.

Reply:
removed